dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -617,7 +617,7 @@ rm -f "${INSTALLER_RENDER_PATH}"
 # Wait for objects to be ready
 # =========================
 for item in deployment.apps/blobserve deployment.apps/content-service deployment.apps/dashboard deployment.apps/ide-metrics deployment.apps/ide-proxy deployment.apps/ide-service deployment.apps/image-builder-mk3 deployment.apps/minio deployment.apps/node-labeler deployment.apps/payment-endpoint deployment.apps/proxy deployment.apps/public-api-server deployment.apps/redis deployment.apps/server deployment.apps/spicedb deployment.apps/usage deployment.apps/ws-manager deployment.apps/ws-manager-bridge deployment.apps/ws-proxy statefulset.apps/messagebus statefulset.apps/mysql statefulset.apps/openvsx-proxy daemonset.apps/agent-smith daemonset.apps/fluent-bit daemonset.apps/registry-facade daemonset.apps/ws-daemon; do
-  kubectl --kubeconfig "${PREVIEW_K3S_KUBE_PATH}" --context "${PREVIEW_K3S_KUBE_CONTEXT}" rollout status "${item}"
+  kubectl --kubeconfig "${PREVIEW_K3S_KUBE_PATH}" --context "${PREVIEW_K3S_KUBE_CONTEXT}" --namespace "${PREVIEW_NAMESPACE}" rollout status "${item}"
```

Comment:
was seeing not-found errors here, setting the namespace fixed that. Think the namespace gets updated and doesn't point to default at this point?

Reply:
:100: same change was also made [here](https://github.com/gitpod-io/gitpod/pull/16908/commits/bac81b9612808e85301ae5d35b87c17f5710f0f5#diff-590a17333c6831c6ff532edbab0b818ee3c49fb033d163722552bbbfafd31a8cR627), to fix triggering workspace integration tests in werft.

@vulkoingim @mads-hartmann can we trouble you for reviews?